docs/reference/commandline/swarm_init.md

```diff
@@ -22,6 +22,7 @@ Initialize a swarm
 
 Options:
       --advertise-addr value            Advertised address (format: <ip|interface>[:port])
+      --autolock                        Enable or disable manager autolocking (requiring an unlock key to start a stopped manager)
```

Comment:
This one is for `init`, so it can only "enable" it, so this can be simply;

```
Enable manager autolocking (requiring an unlock key to start a stopped manager)
```